### PR TITLE
Updated ppv_cube.py to avoid deprecated function 'update_all_headers'

### PR DESCRIPTION
**Pull request by spectram**

### CODE REVIEW

File: yt_astro_analysis/ppv_cube/ppv_cube.py [hunk 1]
```diff
@@ -418,6 +418,7 @@ def _intensity(field, data):
                     / self.dv_cgs
                 )
                 w[w < 0.0] = 0.0
+                # print(self.current_v, w)
```
Comment (neutrinoceros) [inline yt_astro_analysis/ppv_cube/ppv_cube.py:421]: can you clean this please ?

**Review by neutrinoceros — COMMENTED** [review]
Nice catch. This function's deprecation is actually already over, so this counts as a bug fix, thanks !

File: yt_astro_analysis/ppv_cube/ppv_cube.py (hunk 1 above)
Comment (spectram) [reply yt_astro_analysis/ppv_cube/ppv_cube.py]: Done

**Review by neutrinoceros — APPROVED** [review]
thank you !

**Pull request merged**